LDM-151.tex

```diff
@@ -11,7 +11,7 @@
 \interfootnotelinepenalty=10000
 
 
-\setcounter{secnumdepth}{5}
+\setcounter{secnumdepth}{3}
```

Comment:
The class default is 5, and I wonder if it's five because we inherited it from LDM-151. I will say that the LSST document standard (LPM-51 and references therein) seems to support a depth of 7 for section numbering so it's not clear that we should restrict this to 3.